### DISCUSSION TITLE
GraphiQL dark theme

### DISCUSSION THREAD

**Discussion by lvond** [graphql-dotnet/server#1173] [Q&A] [answered]
Just switched IDE from Playground to GraphiQL (latest `GraphQL.Server.Ui.GraphiQL`) and the css styling looks slightly broken on my side when using dark theme.

- result panel still light
- some texts are hard to read due to low contrast

Am I doing something wrong or do you have same experience?

graphql-dotnet
![image](https://github.com/user-attachments/assets/58038a3c-2be0-401e-b53a-542a735abcb9)

graphiql live demo
![image](https://github.com/user-attachments/assets/a14dc32e-b72e-41d0-bc27-5f5dd2857bbe)

**Comment by Shane32** [comment, marked as answer]
It's loading a "GraphiQL Explorer" extension which was designed for a much older version of GraphiQL and is not compatible with GraphiQL 3.x.  There is an option to disable the extension, but there's no code written tied to that property -- so it still gets loaded anyway.

See:
- https://github.com/onegraph/graphiql-with-extensions
- https://github.com/OneGraph/graphiql-explorer

**Reply by Shane32** [reply]
See:
- https://github.com/graphql-dotnet/server/pull/1174

**Reply by Shane32** [reply]
After this PR it will render correctly.  For an immediate fix, you may configure the GraphiQLOptions.PostConfigure property to remove the offending graphiql-with-extensions references from the rendered html.
